tallmanbrew.md

```diff
@@ -0,0 +1,6 @@
+# GitHub Student
+
+* Name: Tallman
+* Location: Indiana
+* UserName: @tallmanbrew
+* twitter: @tallmanbrew
```

Comment:
@tallmanbrew Will you please tell us all the most embarrassing thing that ever happened to you? Or tell us how many fingers you have.